lib/experimental/blocks.php

```diff
@@ -168,14 +171,14 @@ function gutenberg_render_block_connections( $block_content, $block, $block_inst
 				continue;
 			}
 
-			$tags  = new WP_HTML_Tag_Processor( $block_content );
-			$found = $tags->next_tag(
-				array(
-					// TODO: In the future, when blocks other than Paragraph and Image are
-					// supported, we should build the full query from CSS selector.
-					'tag_name' => $block_type->attributes[ $attribute_name ]['selector'],
-				)
-			);
+			$selectors = explode( ',', $block_type->attributes[ $attribute_name ]['selector'] );
+			$found     = false;
+			while ( ! $found && count( $selectors ) > 0 ) {
+				$tags = new WP_HTML_Tag_Processor( $block_content );
+				// TODO: In the future, when blocks are supported,
```

Comment:
I'm guessing a bit at the meaning of the comment
```suggestion
				// TODO: In the future, to support connecting more block attributes,
```